AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Core/SQLStatementTests.swift

```diff
@@ -352,6 +352,135 @@ class SQLStatementTests: XCTestCase {
+                                        sort: .ascending(Post.keys.id))
+        let expectedStatement = """
+        select
+          "root"."id" as "id", "root"."content" as "content", "root"."createdAt" as "createdAt",
+          "root"."draft" as "draft", "root"."rating" as "rating", "root"."status" as "status",
+          "root"."title" as "title", "root"."updatedAt" as "updatedAt"
+        from Post as "root"
+        order by id ASC
+        """
+        XCTAssertEqual(statement.stringValue, expectedStatement)
+    }
+
+    // MARK: - Select Statements Sorts
+
+    /// - Given: a `Model` type and two `QuerySortInput`s
+    /// - When:
+    ///   - the model is of type `Post`
+    ///   - the sort should be `id` of descending order and `createdAt` of descending order
+    /// - Then:
+    ///   - check if the generated SQL statement is valid
+    ///   - check if the statement contains the correct `order by` and `ascending`
+    func testSelectStatementWithTwoFieldsSort() {
+        let statement = SelectStatement(from: Post.self,
+                                        sort: .by(.ascending(Post.keys.id),
+                                                  .descending(Post.keys.createdAt)))
+        let expectedStatement = """
+        select
+          "root"."id" as "id", "root"."content" as "content", "root"."createdAt" as "createdAt",
+          "root"."draft" as "draft", "root"."rating" as "rating", "root"."status" as "status",
+          "root"."title" as "title", "root"."updatedAt" as "updatedAt"
+        from Post as "root"
+        order by id ASC, createdAt DESC
+        """
+        XCTAssertEqual(statement.stringValue, expectedStatement)
+    }
+
+    // MARK: - Select Statements Where and Sort
+
+    /// - Given: a `Model` type, a `QueryPredicate`and  a `QuerySortInput`
+    /// - When:
+    ///   - the model is of type `Post`
+    ///   - the sort should meet predicate condition be `id` of descending order
+    /// - Then:
+    ///   - check if the generated SQL statement is valid
+    ///   - check if the statement contains the correct `where` statement, `order by` and `ascending`
+    func testSelectStatementWithPredicateAndSort() {
+        let statement = SelectStatement(from: Post.self,
+                                        predicate: Post.keys.rating > 4,
+                                        sort: .descending(Post.keys.id))
+        let expectedStatement = """
+        select
+          "root"."id" as "id", "root"."content" as "content", "root"."createdAt" as "createdAt",
+          "root"."draft" as "draft", "root"."rating" as "rating", "root"."status" as "status",
+          "root"."title" as "title", "root"."updatedAt" as "updatedAt"
+        from Post as "root"
+        where 1 = 1
+          and "root"."rating" > ?
+        order by id DESC
+        """
+        XCTAssertEqual(statement.stringValue, expectedStatement)
+    }
+
+    // MARK: - Select Statements Where and Sort
```

Comment:
It seems odd to have one test per section, perhaps just have a single 
`// MARK: - Select Statements With Sort` at the very top of your new tests

Reply:
OK.